### PR TITLE
modifying security controls for client project iam package

### LINKED ISSUE #759: Adding security controls to the client project iam package
Security controls in client-project-iam require an update

### PR DESCRIPTION
**Pull request by anoopsidhu-ssc**
closes #759

### CODE REVIEW

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

File: examples/landing-zone-v2/configconnector/tier3/client-project-iam/iam-tier4-sa.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #########
-# AU-9, AC-3 - IAM Partial Policy that binds tier4 namespace service account to the required minimum project scoped roles to deploy allowed resources
+# AC-3, AC-3(7), AC-16(2) - IAM Partial Policy that binds tier4 namespace service account to the required minimum project scoped roles to deploy allowed resources
```
Comment (alaincormier-ssc) [inline examples/landing-zone-v2/configconnector/tier3/client-project-iam/iam-tier4-sa.yaml:15]: it should also be updated on line 26

File: examples/landing-zone-v2/configconnector/tier3/client-project-iam/iam-users.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #########
-# AU-9, AC-3 - IAM Partial Policy that binds users to the required minimum project scoped roles to perform duties
+# AC-3, AC-3(7), AC-16(2) - IAM Partial Policy that binds users to the required minimum project scoped roles to perform duties
```
Comment (alaincormier-ssc) [inline examples/landing-zone-v2/configconnector/tier3/client-project-iam/iam-users.yaml:15]: it should also be updated on line 26

File: examples/landing-zone-v2/configconnector/tier3/client-project-iam/iam-tier4-sa.yaml (hunk 1 above)
Comment (anoopsidhu-ssc) [reply examples/landing-zone-v2/configconnector/tier3/client-project-iam/iam-tier4-sa.yaml:15]: good catch alain thanks

**Review by cartyc — APPROVED** [review]

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Pull request merged**